src/pudl/output/ferc1.py

```diff
@@ -3044,25 +3098,109 @@ def out_ferc1__yearly_rate_base(
+      and null tags across the component tags.
+
+    Args:
+        rate_base_df: full table of rate base data.
+        tag_col: column with the tags that contains unlabeled values
+            (ex: null or total).
+    """
+    # this works for both the utility_type and in_rate_base tags because
+    # for both tags unlabeled is represented as a null and/or a "total"
+    unlabeled_mask = (rate_base_df[tag_col] == "total") | rate_base_df[tag_col].isnull()
+    ratio_idx = ["report_year", "utility_id_ferc1"]
+    ratio_df = get_tag_col_ratio(
+        # remove the unlabeled records because total is the value
+        # we want to break_down so we can't have it in the columns to sum up
+        rate_base_labeled_df=rate_base_df[~unlabeled_mask],
+        ratio_idx=ratio_idx,
+        tag_col=tag_col,
+    )
+    unlabeled_break_down_df = (
+        pd.merge(
+            rate_base_df[unlabeled_mask],
+            ratio_df,
+            on=ratio_idx,
+            how="left",
+            validate="m:m",
+            suffixes=("_unlabeled", ""),
+        )
+        .assign(
+            ending_balance=lambda x: x[f"ratio_{tag_col}"].fillna(1) * x.ending_balance,
+        )
+        .assign(**{f"is_break_down_{tag_col}": True})
+        .drop(columns=[f"ratio_{tag_col}", f"{tag_col}_unlabeled"])
+        # this automatially gets converted to a pandas Float64 which
+        # results in nulls from any sum.
+        .astype({"ending_balance": float})
+    )
+    rate_base_broken_down_df = pd.concat(
         [
-            _out_ferc1__detailed_balance_sheet_assets[
-                _out_ferc1__detailed_balance_sheet_assets.tags_in_rate_base.isin(
-                    ["yes", "partial"]
-                )
-            ],
-            _out_ferc1__detailed_balance_sheet_liabilities[
-                _out_ferc1__detailed_balance_sheet_liabilities.tags_in_rate_base.isin(
-                    ["yes", "partial"]
-                )
-            ].assign(ending_balance=lambda x: -x.ending_balance),
-            cash_working_capital,
+            rate_base_df[~unlabeled_mask].assign(**{f"is_break_down_{tag_col}": False}),
+            unlabeled_break_down_df,
         ]
-    ).sort_values(by=["report_year", "utility_id_ferc1", "table_name"], ascending=False)
-    # note: we need the `tags_in_rate_base` column for these checks
-    check_tag_propagation_compared_to_compiled_tags(
-        in_rate_base, "in_rate_base", _out_ferc1__detailed_tags
     )
-    check_for_correction_xbrl_factoids_with_tag(in_rate_base, "in_rate_base")
-    return in_rate_base
+    if not np.isclose(
+        new_balance := rate_base_broken_down_df.ending_balance.sum(),
+        old_balance := rate_base_df.ending_balance.sum(),
+    ):
+        logger.warning(
+            f"{tag_col}: New ending balance is not the same as the old ending balance: "
+            f"{old_balance=}, {new_balance=}"
+        )
+    return rate_base_broken_down_df.convert_dtypes()
```

Comment:
good point on the log level. i think i converted it to a warning while debugging. i will swap out.

this could only happen before the concatenation if we were testing this on only the records that were unlabeled vs the disaggregated. i am partial to keeping it after the concatenation bc it answers the question "is the total ending balance the same before and after we disaggregated some of the records?"